src/main/java/seedu/address/ui/TransactionCard.java

```diff
@@ -37,9 +41,27 @@ public class TransactionCard extends UiPart<Region> {
     public TransactionCard(Transaction transaction, int displayedIndex) {
         super(FXML);
         this.transaction = transaction;
-        transactionId.setText(Long.toString(transaction.getId().value));
-        description.setText(transaction.getDescription().value);
-        amount.setText(Double.toString(transaction.getAmount().value));
-        employeeId.setText(Integer.toString(transaction.getEmployeeId().value));
+
+        String transactionLabel = "Transaction ID: ";
+        String transactionIdText = Long.toString(transaction.getId().value);
+        transactionId.setText(transactionLabel + transactionIdText);
+
+        String amountLabel = "Amount: $";
+        String amountText = Double.toString(transaction.getAmount().value);
+        amount.setText(amountLabel + amountText);
+
+        String employeeIdLabel = "Employee ID: ";
+        String employeeIdText = Integer.toString(transaction.getEmployeeId().value);
+        employeeId.setText(employeeIdLabel + employeeIdText);
+
+        String descriptionLabel = "Description: ";
+        String descriptionText = transaction.getDescription().value;
+        description.setText(descriptionLabel + descriptionText);
+
+
+        DateTimeFormatter formatter = DateTimeFormatter.ofPattern("dd/MM/yyyy HH:mm");
```

Comment:
Perhaps can obtain the DATETIME_FORMAT from DateTime class?

Reply:
I have removed the `formatter`, and chosen to use `getDateTime` directly from the `transaction`.